src/Pagination/Pagination.tsx

```diff
@@ -19,7 +20,7 @@ export default function Pagination() {
         pageCount={pageCount}
         breakLabel="..."
         onPageChange={({ selected }) => {
-          dispatch({ type: 'handleOffset', payload: selected }); // TODO: fix this 'selected' variable
+          dispatch(handleOffsetAction(selected)); // TODO: fix this 'selected' variable
```

Comment:
Please, extract this handler from JSX.

const handlePageChange = ({ selected }) => {
    dispatch(handleOffsetAction(selected));
}

Btw, can you get rid of this TODO?

Reply:
Moved this handler from JSX. The TODO has been terminated.